.joss/paper.md

```diff
@@ -0,0 +1,90 @@
+LSS tackles this problem by making as many GLMs as there are trials.
+For each trial, a GLM is created where one predictor is the trial of interest and the other predictor is the combination of the other trials.
+This reduces the amount of overlap (or more accurately correlation) between predictors, leading to more reliable individual beta estimates.
+NiBetaSeries currently implements LSS making it an ideal candidate for experiments with trials that occur closer together (e.g. 3-7 seconds apart on average).
+
+The second question of what an atlas parcellation is will hopefully be easier to explain.
+The brain can be cut up in many ways, either via anatomical landmarks, by which regions are activated by a task of theoretical interest, by how the brain organizes itself when not engaged in any external task, or a number of other creative methods.
+However the brain is sliced up, we need to mark all the selected voxels within a brain region with the same number to uniquely identify the region.
+A voxel is shortened from the term volumetric pixel, unlike typical pictures our brain images are 3 dimensional, so in order to account for the volume we use 3D voxels instead of 2D pixels.
+An image with all the regions uniquely labelled is considered an atlas parcellation for the purposes of NiBetaSeries.
+The unique regions will be used to average across the voxels containing beta estimates for each trial.
+The averaging of beta estimates is repeated over all the trials resulting in a list of averaged beta estimates for that region.
+The list of betas for each region can then be correlated with one another to measure the degree of synchrony between regions.
+
+# Overview
+
+NiBetaSeries presents as a command line utility written in python.
+The primary way for users to interact with NiBetaSeries is by typing `nibs` in the command line.
+The basic workflow of NiBetaSeries follows these steps (the files can be found in the `workflows` directory):
+1) `base.py`: read and validate necessary inputs (a minimally processed fMRI file, a brain mask, an events file specifying the onsets, duration, and the trial type, an atlas parcellation, and a table connecting the numbers in the atlas parcellation with names of the regions)
+2) `model.py`: construct and execute GLMs using LSS (with additional confound predictors optionally added) generating betaseries (a list of betas for each voxel)
+3) `analysis.py`: apply atlas parcellation to data averaging betas within regions for each trial
+4) `analysis.py`: correlate each region's list of betas with every other region.
+5) `analysis.py`: r-z transform the correlations and output a symmetric correlation matrix in a `tsv` file.
+
+The correlation matrix from NiBetaSeries can be used for graph theoretical analysis, specific region-region correlations across sessions or across different trial types, and presumably other analytic strategies.
```

Comment:
the phrase "across sessions" could be misunderstood to mean correlations between two sessions of data, i'm assuming you mean within sessions for graph?